src/Compilers/Core/Portable/PEWriter/MetadataWriter.cs

```diff
@@ -2872,6 +2882,36 @@ private int[] SerializeMethodBodies(BlobBuilder ilBuilder, PdbWriter nativePdbWr
             return bodyOffsets;
         }
 
+        private int SerializeThrowNullMethodBody(MethodBodyStreamEncoder encoder, ref UserStringHandle mvidStringHandle, ref Blob mvidStringFixup)
```

Comment:
>ref UserStringHandle mvidStringHandle, ref Blob mvidStringFixup [](start = 82, length = 63)

It looks like these parameters are never going to be used in a meaningful way. We can use locals instead and assert that ```WriteInstructions``` doesn't change them.